core/controllers/editor.py

```diff
@@ -16,15 +16,11 @@
 
 """Controllers for the editor view."""
 
+import StringIO
```

Comment:
It seems a bit odd to me to have uppercase above lowercase when sorting, instead of treating this similar to stringIO. Is there a setting that can be used to ignore case? If not, that's fine.

Reply:
I could not find a way around this. There was no setting pertaining to this.